sdk/python/feast/cli.py

```diff
@@ -1018,5 +1022,114 @@ def permission_describe(ctx: click.Context, name: str):
     )
 
 
+@feast_permissions_cmd.command(name="check")
+@click.pass_context
+def feast_permissions_check_command(ctx: click.Context):
+    """
+    Validate the permissions configuration
+    """
+    from tabulate import tabulate
+
+    all_unsecured_table: list[Any] = []
+    store = create_feature_store(ctx)
+    permissions = store.list_permissions()
+    objects = cli_utils.fetch_all_feast_objects(
+        store=store,
+    )
+
+    print(
+        f"{Style.BRIGHT + Fore.RED}The following resources are not secured by any permission configuration:{Style.RESET_ALL}"
+    )
+    for o in objects:
+        cli_utils.handle_permissions_check_command(
+            object=o, permissions=permissions, table=all_unsecured_table
+        )
+    print(
+        tabulate(
+            all_unsecured_table,
+            headers=[
+                "NAME",
+                "TYPE",
+            ],
+            tablefmt="plain",
+        )
+    )
+
+    all_unsecured_actions_table: list[Any] = []
+    print(
+        f"{Style.BRIGHT + Fore.RED}The following actions are not secured by any permission configuration (Note: this might not be a security concern, depending on the used APIs):{Style.RESET_ALL}"
+    )
+    for o in objects:
+        cli_utils.handle_permissions_check_command_with_actions(
+            object=o, permissions=permissions, table=all_unsecured_actions_table
+        )
+    print(
+        tabulate(
+            all_unsecured_actions_table,
+            headers=[
+                "NAME",
+                "TYPE",
+                "UNSECURED ACTIONS",
+            ],
```

Comment:
isn't it redundant information to display the same info twice?

Reply:
It's not the same info: in `feast permissions check` the first table has all the resources with no permissions configured matching type, name and tags
The second table instead shows the non-secured actions for resources that have configured permissions for its type, name and tag.